crates/graphql/src/schema/objects/wallet.rs

```diff
@@ -20,16 +21,15 @@ impl Wallet {
 
 #[graphql_object(Context = AppContext)]
 impl Wallet {
-    pub fn address(&self) -> &str {
+    pub fn address(&self) -> &PublicKey<Wallet> {
         &self.address
     }
 
     pub fn bids(&self, ctx: &AppContext) -> FieldResult<Vec<Bid>> {
         let db_conn = ctx.db_pool.get()?;
-
         let rows: Vec<models::Bid> = bids::table
-            .select(bids::all_columns)
-            .filter(bids::bidder_address.eq(&self.address))
+            .select(bids::all_columns)      // How to avoid cloning?
+            .filter(bids::bidder_address.eq::<String>(self.address.clone().into()))
```

Comment:
PublicKey should be usable here without an Into

Reply:
I get this when I remove the eq::<String> (just eq) and using the PublicKey directly
```
the trait bound `PublicKey<Wallet>: diesel::expression::Expression` is not satisfied
required because of the requirements on the impl of `diesel::expression::AsExpression<Text>` for `PublicKey<Wallet>`
```

Reply:
I'll look into this.